### PR TITLE
additional matching functions in z_room, z_camera, and En_Zl3

### PR DESCRIPTION
**Pull request by petrie911**
Matched a function in each of these and made improvements to the matchings in z_room. I also have duplicates of roman's tubo_trap and po sisters. Merge that one first.

### CODE REVIEW

**Review by Roman971 — APPROVED** [review]

File: src/overlays/actors/ovl_En_Zl3/z_en_zl3.c [hunk 1]
```diff
@@ -234,20 +234,19 @@ void func_80B53974(EnZl3* this, u8 arg1) {
     this->unk_3C8 = arg1;
 }
 
-#ifdef NON_MATCHING
```
Comment (ethteck) [inline src/overlays/actors/ovl_En_Zl3/z_en_zl3.c:237]: nice! 🎊

File: include/z64math.h [hunk 2]
```diff
@@ -95,7 +95,7 @@ typedef struct {
 /**
  * Trig macros
 */
-#define DEGF_TO_BINANG(degreesf) (s16)(degreesf * 182.04167f + 0.5f)
+#define DEGF_TO_BINANG(degreesf) (s16)(degreesf * 182.04167f + .5f)
```
Comment (fig02) [inline include/z64math.h:98]: im worried that this specific fix is only relevant for camera. if it starts breaking stuff outside of camera then it should probably be a local macro

Comment (petrie911) [reply include/z64math.h:98]: I believe Krim made it for camera. I don't know if anything else even uses that.

Comment (fig02) [reply include/z64math.h:98]: yea he did, but it was put in z64math with the intention of being used everywhere i guess. just something to keep in mind

**Review by fig02 — APPROVED** [review]

File: include/z64math.h (hunk 2 above)
Comment (petrie911) [reply include/z64math.h:98]: speaking of macros in math, BINANG_SUB should probably be SUB16

**Pull request merged**